packages/contracts/contracts/ArgoBridgeV1.sol

```diff
@@ -169,7 +170,8 @@ contract ArgoBridgeV1 is AccessControl {
 
     function withdrawBridgeFees() public onlyRole(DEFAULT_ADMIN_ROLE) {
         uint256 balance = address(this).balance;
-        payable(msg.sender).transfer(balance);
+        (bool success, ) = payable(msg.sender).call{value: balance}("");
+        require(success, "Transfer failed");
```

Comment:
Why?

Reply:
(]https://detectors.auditbase.com/using-call-over-transfer-payable-addresses

Reply:
I don't see the problem described there being an issue for us and I'd rather have guaranteed reentrancy protection